devices/Max1704x/category.txt

```diff
@@ -0,0 +1,2 @@
+power
+battery
```

Comment:
battery does not exist in the list, see https://github.com/nanoframework/nanoFramework.IoT.Device/blob/develop/src/device-listing/Program.cs

but power exists and should be enough

Reply:
done